src/subprocess/runner.ts

```diff
@@ -217,42 +184,60 @@ class Reporter extends reporters.Base {
       event: 'sync',
     });
 
+    assert(event);
```

Comment:
Looks like a premature optimization. Let's either add a test for it or remove this.